### DISCUSSION TITLE
Position does not work when vueflow has a parent node

### DISCUSSION THREAD

**Discussion by homeless2010** [bcakmakoglu/vue-flow#1720] [Q&A]
Why can't the node be positioned accurately when there is a parent node? Is there any way to position the node accurately?

**Comment by bcakmakoglu** [comment]
To get a proper response to a question you should at least
* Describe what you're trying to do and what the expected outcome is supposed to be or look like
* Describe what hasn't worked
* Provide a reproduction of the issue

Writing two sentences like "why isnt XYZ working?????" isn't gonna cut it.
I'm closing this post, feel free to try again.
